manifests/init.pp

```diff
@@ -9,7 +9,7 @@
   Optional[Boolean] $test = undef,
   Optional[Stdlib::HTTPUrl] $relay_api_url = undef,
   Optional[Sensitive[String]] $relay_connection_token = undef,
-  Optional[Variant[Array[Sensitive[String]], Sensitive[String]]] $relay_trigger_token = undef,
+  Optional[Variant[Sensitive[Array[String]], Sensitive[String]]] $relay_trigger_token = undef,
```

Comment:
Should both `Array[Sensitive[...]]` and `Sensitive[Array[...]]` be allowed? Or is that impossible with EYAML?

Reply:
@impl It's actually not related to eyaml, but the way hiera lookups work. If the key `relay::relay_trigger_token` is looked up, it will return an array in case there are multiple tokens. The whole array needs to be Sensitive.